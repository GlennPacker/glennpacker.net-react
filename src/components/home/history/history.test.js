import React from 'react';
import { shallow } from 'enzyme';

import History from './index';

describe('history', () => {
    let wrapper
    beforeAll(() => {
        wrapper = shallow(<History />);
    })

    it('should render', () => {
        expect(wrapper).not.toBeNull();
    })

    it('should render correctly compared to snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    })
})