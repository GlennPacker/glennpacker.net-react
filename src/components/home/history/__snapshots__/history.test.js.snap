// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`history should render correctly compared to snapshot 1`] = `
<div
  className="card"
>
  <Style
    rules={
      Object {
        ".history-card-body": Object {
          "minHeight": 450,
        },
        "ul": Object {
          "listStyle": "none",
          "paddingLeft": 0,
        },
      }
    }
    scopeSelector=""
  />
  <div
    className="card-header bg-success text-white"
  >
    History
  </div>
  <div
    className="card-body history-card-body"
  >
    <p
      className="card-text"
    >
      With over 25 years experience in the IT industry from support, training and development.
    </p>
    <Connect(HistoriesContainer) />
  </div>
</div>
`;
